cmd/validate/__snapshots__/image_test.snap

```diff
@@ -0,0 +1,8 @@
+
+[Test_ValidateImageCommandYAMLPolicyFile/spec - 1]
+{"success":true,"components":[{"name":"Unnamed","containerImage":"registry/image:tag","source":{},"success":true}],"key":"-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAECBtqKHcvxYkGx7ZXqps3nrYS+ZSA\nmh3m1MZfTGlnr2oN0z+sBWEC23s4RkVSXkEydI6SLYatUtJK8OmiBRS+Xw==\n-----END PUBLIC KEY-----\n","policy":{"description":"My custom enterprise contract policy configuration","sources":[{"policy":["quay.io/hacbs-contract/ec-release-policy:latest"]}],"configuration":{"exclude":["not_useful","test:conftest-clair"],"include":["always_checked","@salsa_one_collection"]},"publicKey":"-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAECBtqKHcvxYkGx7ZXqps3nrYS+ZSA\nmh3m1MZfTGlnr2oN0z+sBWEC23s4RkVSXkEydI6SLYatUtJK8OmiBRS+Xw==\n-----END PUBLIC KEY-----\n"},"ec-version":"development","effective-time":"1970-01-01T00:00:00Z"}
```

Comment:
Not sure if it makes sense here, but consider using ` ${known_PUBLIC_KEY}` `${TIMESTAMP}` etc  to make this snap file.